assets/src/edit-story/components/IconSwitch/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+
+const CheckBoxInput = styled.input.attrs( { type: 'checkbox' } )`
+	position: absolute;
+	opacity: 0;
+`;
+
+const MarkSpan = styled.span`
+	position: absolute;
+	top: 0;
+	left: 0;
+	height: ${ ( { height } ) => height }px;
+	width: ${ ( { width } ) => width }px;
```

Comment:
Couldn't this just be `width: 100%; height: 100%;` so you don't need to define it twice as it's already set on the label? Also I'd like to avoid the absolute positioning actually. Can't you change it to `display:block` and avoid it? Maybe you can even do `width: inherit; height: inherit;`. Please see how this CSS can become simpler while still working correctly. You might also change the display type of the parent label to flexbox to avoid the annoying padding-because-of-line-height issue.

Reply:
okay